cob_twist_controller/include/cob_twist_controller/augmented_solver.h

```diff
@@ -19,30 +21,11 @@
 * @ingroup KinematicFamily
 */
 
-struct AugmentedSolverParams {
-    int damping_method;
-    double eps;
-    double damping_factor;
-    double lambda0;
-    double wt;
-    bool JLA_active;
-    bool enforce_limits;
-    double tolerance;
-    bool base_compensation;
-    bool base_active;
-    double base_ratio;
-
-};
 
-enum DampingMethodTypes {
-    MANIPULABILITY = 0,
-    CONSTANT = 1,
-    TRUNCATION = 2
-};
 
 
 
-class augmented_solver
+class AugmentedSolver
```

Comment:
This is a general question:
I never really liked the wording of ```augmented_solver```/```AugmentedSolver``` as it only covers one aspect (i.e. augmenting the Jacobian by rows and columns) and not the whole aspect of what we are doing here.
(Its just in there from the ancient times where everything got started...)

What about using ```DifferentialKinematicsSolver```?